TESTPLAN

```diff
@@ -55,7 +55,9 @@ Main screen:
 
       <l> - enter LSOF screen.
 
-      <m>, <n>, <o>, <p> - do nothing.
+      <m>, <n>, <o> - do nothing.
+
+      <p> - hide/show program path.
```

Comment:
Usually subject to another PR ;-)

Reply:
Came in from the hishamhm/htop#510 one, I shall accept no blame for poking a dead test script :-)

Reply:
Just put it in a separate commit and I'm totally fine with this change.

My complaint is basically about mixing unrelated changes in the same commit.

Reply:
Spliced it out to c44b2ec